system/Database/BaseBuilder.php

```diff
@@ -1151,7 +1151,7 @@ protected function _like($field, string $match = '', string $type = 'AND ', stri
 
         foreach ($keyValue as $k => $v) {
             if ($insensitiveSearch === true) {
-                $v = strtolower($v);
+                $v = mb_strtolower($v);
```

Comment:
```suggestion
                $v = mb_strtolower($v, 'UTF-8');
```
It seems better to include the second parameter ('UTF-8') in `mb_strtolower` to ensure proper handling of multibyte characters.

Reply:
Good point.